deploy/docker/start.sh

```diff
@@ -1,47 +1,41 @@
 #!/bin/sh
-set -e
+set -e -u
 
 emqx_exit(){
-    # tail erlang.log.*
-    erlang_log=$(echo $(ls -t /opt/emqx/log/erlang.log.*) | awk '{print $1}')
-    num=$(sed -n -e '/LOGGING STARTED/=' ${erlang_log} | tail -1)
-    tail -n +$((num-2)) ${erlang_log}
+    # tail emqx.log.*
+    num=$(sed -n -e '/LOGGING STARTED/=' /opt/emqx/log/emqx.log | tail -1)
+    tail -n +$((num-2)) /opt/emqx/log/emqx.log
 
     echo "['$(date -u +"%Y-%m-%dT%H:%M:%SZ")']:emqx exit abnormally"
     exit 1
 }
 
 ## EMQ Main script
 
-# Start and run emqx, and when emqx crashed, this container will stop
+# Set log.rotation=off in etc/emqx.conf
+sed -i -r 's/log.rotation = (.*)/log.rotation = off/1' /opt/emqx/etc/emqx.conf
 
+# Start and run emqx, and when emqx crashed, this container will stop
 /opt/emqx/bin/emqx start || emqx_exit
```

Comment:
maybe use `trap` in the beginning of this script instead of calling the function explicitly.
 `trap "emqx_exit" EXIT`

Reply:
The "&" symbol is used here because I need to print the log and check the status of emqx. Do you have any better suggestions? I don’t know much about the `trap` command